mllib/src/main/scala/org/apache/spark/ml/clustering/KMeans.scala

```diff
@@ -258,6 +290,27 @@ class KMeans @Since("1.5.0") (
   @Since("1.5.0")
   def setSeed(value: Long): this.type = set(seed, value)
 
+  /** @group setParam */
+  @Since("2.0.0")
+  def setInitialModel(value: KMeansModel): this.type = set(initialModel, value)
```

Comment:
We cannot sure whether users will setK after this action. Say, `new KMeans().setInitialModel(xxx).setK(anotherK)` would helpless.